astrobee_api/scripts/genBaseRobot.py

```diff
@@ -1,24 +1,38 @@
 #!/usr/bin/env python
+# Copyright (c) 2017, United States Government, as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+#
+# All rights reserved.
+#
+# The Astrobee platform is licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
 
 """
-A library and command-line tool for generating base robot java classes/files
-from an XPJSON schema.
+Generate base robot java classes/files from an XPJSON schema.
 """
 
-import os
-import sys
-import re
+import argparse
 import logging
 
-# hack to ensure xgds_planner2 submodule is at head of PYTHONPATH
-astrobee_root = os.getenv('SOURCE_PATH', (os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))))))
-sys.path.insert(0, os.path.join(astrobee_root, 'astrobee', 'commands', 'xgds_planner2'))
-sys.path.insert(0, os.path.join(astrobee_root, 'scripts', 'build'))
+# modify PYTHONPATH to enable xpjsonAstrobee import
+import astrobee_api_util
+
+# isort: split
 
-from xgds_planner2 import xpjson
```

Comment:
This was removed from all 3 scripts and wasn't added to the astrobee api util script. Was it not needed to begin with?

Reply:
The `xpjson` import was never used locally in those scripts, although it was and still is used indirectly via the `xpjsonAstrobee` wrapper.

For context, I found several unnecessary imports using `pylint`, removed them, and tested the edited version before submitting the PR.